### ISSUE TITLE
Is the validation set only for RGB-D task，not for RGB-T task?

### ISSUE THREAD

**Issue by Karry-Wu** [LitterMa-820/P3Net#2]

**Comment by LitterMa-820** [comment]
The validation set takes a small random portion of the dataset and you can divide it up yourself. Or just delete the validation part of the training code, which will not affect the model training results.
